service/provider/etcd/etcd.go

```diff
@@ -1,94 +0,0 @@
-package etcd
```

Comment:
I removed etcd provider completely, because it's annoying with it's old deps and etcd dependency in general. I could split in separate PR, but laziness... I can do if important.